package.json

```diff
@@ -58,6 +59,7 @@
   "private": true,
   "workspaces": [
     "packages/ava",
-    "packages/js"
+    "packages/js",
```

Comment:
Just caught this, but since `js` is a dep of `ava` it should come first in the list.

Reply:
Interesting. Why hasn't that caused problems yet?

Reply:
Because we haven't made significant changes to `js` so the already built files don't break anything in ava.  However, try deleting the `lib` directory and it will be an issue.

Reply:
Gotcha, I created https://github.com/near/runner-js/pull/73 for this issue